### DISCUSSION TITLE
Welcome to degoogle Discussions!

### DISCUSSION THREAD

**Discussion by tycrek** [tycrek/degoogle#403] [General]
## Discussions is replacing `Other` Issues

Thanks to new Issue Forms introduced in #402, the `Other` Issue can now be moved to Discussions.

**Comment by acordeddu** [comment]
I'm the owner of Servermx (servermx.com).
We are an independent email service provider borned in 2012.
We utilize open source email software as we share their inspiring principles. Open source allows even smaller companies to compete with big companies. We believe that e-mail, beyond temporary fads, is a basic building block in the world of communications and for this reason we will continue to put all our efforts to raise the level of the e-mail services.
It's possible to get listed?

**Reply by StephenBrown2** [reply]
Certainly! Fill out a [New Alternative Issue](https://github.com/tycrek/degoogle/issues/new?assignees=&labels=new&template=1-new-alternative.yml&title=%5BNEW%5D+Alternative+name+here) and @tycrek can add your information if it meets the guidelines for the list.
